tests/visual/taskboard-column.html

```diff
@@ -23,13 +23,13 @@
                                 <div class="k-taskboard-column-header-text k-text-ellipsis">Column Header</div>
                                 <span class="k-spacer"></span>
                                 <div class="k-taskboard-column-header-actions">
-                                    <button class="k-button k-button-icon k-button-flat">
```

Comment:
both are fine. In fact, in the scss source files we use `.k-button.k-flat` extends `.k-button-flat`